best_practices/configuration.rst

```diff
@@ -101,7 +101,7 @@ to control the number of posts to display on the blog homepage:
     parameters:
         homepage.num_items: 10
 
-If you ask yourself when the last time was that you changed the value of
+If you ask yourself when was the last time you changed the value of
```

Comment:
Ah, I don't love the new one either. The problem is the sentence is *almost* a question, but not *really* a question, so it's funny both ways. So how about something totally different:

```rst
If you've done something like this in the past, it's likely that you've in fact *never* actually needed
to change that value.
```

What do you think @edsonmedina?